Gemfile

```diff
@@ -67,7 +67,8 @@ gem 'bootstrap', '~> 4.0.0'
 gem 'jquery-rails'
 gem 'kramdown', '~> 1.16'
 gem 'rouge', '~> 3.1'
-gem 'simple_form', '~> 3.5'
+gem 'simple_form', github: 'plataformatec/simple_form'
+gem 'validates_timeliness', '~> 4.0'
```

Comment:
Why we need this gem?

Reply:
I just wanted easy and readable validations for the `date`, `time` and `datetime` examples.

```rb
validates :awake,       presence: true, timeliness: { type: :time, before: '12:00' }
validates :first_kiss,  presence: true, timeliness: { type: :datetime, after: '20:00' }
```
https://github.com/m5o/simple_form-bootstrap/blob/14e4c0b97fc0dec2a06874ce50094bbd9fb31deb/app/models/user.rb#L54

Reply:
Sounds good. Thank you for explaining.